pkg/resolvers/doq.go

```diff
@@ -132,3 +132,71 @@ func (r *DOQResolver) Lookup(question dns.Question) (Response, error) {
 func (r *DOQResolver) SetTLSConfig(tlsCfg *tls.Config) {
 	r.tls = tlsCfg
 }
+
+// Lookup1 likes Lookup, but return a list of dns.Msg instead.
+func (r *DOQResolver) Lookup1(question dns.Question) ([]dns.Msg, error) {
```

Comment:
Maybe a name like `LookupMsgs` is more appropriate?

Reply:
Not sure it's worth. `Lookup` is a verb, and `LookupMsgs` sounds like looking up these messages.

The suffix 1 here indicates that this is a variant of `Lookup`, which will return a different kind of result (we documented it in the comment).

For example, you can look at https://pkg.go.dev/github.com/dsnet/try#E1, though they use the suffix to indicate number of return results.